From 9ce944e21115d71f68107462e24cc6b0234a1d5a Mon Sep 17 00:00:00 2001
From: "Ruan, Hongfu" <hongfu.ruan@intel.com>
Date: Tue, 22 Sep 2020 09:48:34 +0800
Subject: [PATCH] update drm interface version

update interface for clearkey, remove widevine which is not part
of caas by default.

Tracked-On: OAM-92991
Signed-off-by: Ruan, Hongfu <hongfu.ruan@intel.com>
---
 groups/device-specific/caas/manifest.xml | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index 7da9fa4..286fcc1 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -176,10 +176,8 @@
             <name>IDrmFactory</name>
             <instance>default</instance>
         </interface>
-        <fqname>@1.1::ICryptoFactory/clearkey</fqname>
-        <fqname>@1.1::IDrmFactory/clearkey</fqname>
-        <fqname>@1.1::ICryptoFactory/widevine</fqname>
-        <fqname>@1.1::IDrmFactory/widevine</fqname>
+        <fqname>@1.3::ICryptoFactory/clearkey</fqname>
+        <fqname>@1.3::IDrmFactory/clearkey</fqname>
     </hal>
     <hal format="hidl">
         <name>android.hardware.camera.provider</name>
-- 
2.17.1

